data/battle_scripts_1.s

```diff
@@ -399,7 +399,6 @@ BattleScript_EffectSkyDrop:
 	attackcanceler
 	ppreduce
 	accuracycheck BattleScript_PrintMoveMissed, ACC_CURR_MOVE
-	attackstring
```

Comment:
Won't removing this make it so Sky Drop will no longer print the "Pokémon used Sky Drop!" string?

Reply:
No, because attackstring is used in BattleScriptFirstChargingTurn.